Makefile

```
@@ -1,10 +1,21 @@
 .DEFAULT_GOAL := up
+.EXPORT_ALL_VARIABLES:
 
 KUBECONFIG := $(shell pwd)/.kubeconfig
+MINI_LAB_FLAVOR := $(or $(MINI_LAB_FLAVOR),mini)
+
+# Default values
+VAGRANT_VAGRANTFILE=Vagrantfile
+DOCKER_COMPOSE_OVERRIDE=
+VAGRANT_MACHINES=machine01 machine02
+
+ifeq ($(MINI_LAB_FLAVOR),cluster-api)
+VAGRANT_MACHINES=machine01 machine02 machine03
+endif
```

Maybe it's good to even fail when a user defines an unknown flavor.